charts/images.yaml

```diff
@@ -423,10 +441,28 @@ images:
   sourceRepository: github.com/istio/istio
   repository: gcr.io/istio-release/proxyv2
   tag: "1.15.3-distroless"
+  labels:
+  - name: 'gardener.cloud/cve-categorisation'
+    value:
+      network_exposure: 'public'
+      authentication_enforced: false
+      user_interaction: 'end-user'
+      confidentiality_requirement: 'low'
+      integrity_requirement: 'high'
+      availability_requirement: 'high'
 - name: istio-istiod
   sourceRepository: github.com/istio/istio
   repository: gcr.io/istio-release/pilot
   tag: "1.15.3-distroless"
+  labels:
+  - name: 'gardener.cloud/cve-categorisation'
+    value:
+      network_exposure: 'protected'
```

Comment:
Is there any communication that's not inside the VPC here. Shouldn't it be `private`.

Reply:
I assumed that `istiod` will also talk to `kube-apiserver` of the cluster, which would be a well-known entity outside the VPC.
WDYT?

Reply:
Yes, true! 
Sorry, I was thinking in the other direction.